source/manual/local-frontend-development.html.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Local frontend development
+parent: "/manual.html"
+layout: manual_layout
+section: Frontend
+type: learn
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2019-07-08
+review_in: 6 months
+---
+
+Make the following changes to your local setup:
+
+In [govuk_app_config], add your local domain to the allowed CORS domains:
+
+```
+  GOVUK_DOMAINS = [
+    '*.publishing.service.gov.uk',
+    # "*.#{ENV['GOVUK_APP_DOMAIN_EXTERNAL'] || ENV['GOVUK_APP_DOMAIN'] || 'dev.gov.uk'}"
+     '*.dev.gov.uk'
+  ].uniq.freeze
+```
+
+You need to change the frontend app you are working on to use the modified [govuk_app_config]. For example in [frontend] update the Gemfile:
+
+```ruyby
+...
+gem 'govuk_app_config', path: '/var/govuk/govuk_app_config'
+...
+gem 'govuk_publishing_components', path: '/var/govuk/govuk_publishing_components'
+...
+```
+
+and set in `development.rb`:
+
+```ruby
+...
+config.assets.debug = false
+...
+```
+
+Update the Gemfile in [static] to point to [govuk_publishing_components] gem:
```

Comment:
This is only relevant if the change you are making to the gem affects static. Most changes shouldn't as they just get pulled in by the frontend app you're running, so it would be enough for most people to update the gem in the frontend app only. It's likely to only be things that we have across the whole site, e.g: cookie banner (and survey banner possibly).

This should be helped if we split the guide up into sections as mentioned in another comment.